utils/Authenticate.js

```diff
@@ -0,0 +1,27 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+/**
+ * A class that handles authentication including generating and verifying user tokens.
+ */
+class Authenticate {
+  /**
+   * Generates user tokens
+   * @param {int} id
+   * @param {string} email
+   * @returns {string} token
+   */
+  static generateToken(id, email) {
+    const token = jwt.sign({
+      userId: id, email
```

Comment:
Please sign the `username` as well @chizzydavid

Reply:
@fortunee I made a few changes to the user model, I was going to run them by you this morning. So I moved a few data-fields including `username` from the User model to the Profile Model. I'd post the details on slack